src/sagemaker/serve/utils/tuning.py

```diff
@@ -216,3 +264,71 @@ def _more_performant(best_tuned_configuration: list, tuned_configuration: list)
             return True
         return False
     return tuned_avg_latency <= best_avg_latency
+
+
+def _more_performant_benchmark(
```

Comment:
Is this new logic added here, or can we use existing logic in tuning.py? Seems  _more_performant exists as well

https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/serve/utils/tuning.py#L207C4-L207C21